calculator.js

```diff
@@ -26,4 +53,9 @@ class Calculator {
 * - Method Chaining
 */
 
+let test = new Calculator();
+// console.log(test.add(5).square(2).subtract(5).multiply(2).divide(8).add(4).squareRoot())
+console.log(test.add(3).circumfrence())
```

Comment:
tidak ada method untuk memberikan hasilnya, misal:
equal() {
  return this.result
}